code/game/machinery/Sleeper.dm

```diff
@@ -110,13 +110,19 @@
 		if (length(inserted_vials) >= max_vials)
 			to_chat(user, "<span class='warning'>[src] cannot hold any more!</span>")
 			return
+		user.visible_message("<span class='notice'>[user] inserts \the [I] into \the [src]</span>", "<span class='notice'>You insert \the [I] into \the [src]</span>")
 		user.temporarilyRemoveItemFromInventory(I)
 		I.forceMove(null)
 		inserted_vials += I
 		ui_update()
 		return
 	. = ..()
 
+/obj/machinery/sleeper/on_deconstruction()
```

Comment:
Not a fix, it's a rebalance
It's not dropping them on purpose

Reply:
@PowerfulBacon did you intend sleepers to not drop their contents when deconstructed?

Reply:
Looks like he forgor

Reply:
Ok